extensions/annotations/src/main/java/io/opentelemetry/extension/annotations/Timed.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.extension.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This annotation creates a {@link io.opentelemetry.api.metrics.LongHistogram Histogram} instrument
+ * observing the duration of invocations of the annotated method or constructor.
+ *
+ * <p>By default, the Histogram instrument will have the following attributes:
+ *
+ * <ul>
+ *   <li><b>class:</b> The fully qualified name of the class whose method is invoked.
+ *   <li><b>method:</b> The name of the annotated method, or "new" of the annotation is on a
+ *       constructor.
```

Comment:
Instead of `class` and `method`, WDYT about using `code.namespace` and `code.function`? These attributes are already in the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/span-general.md#source-code-attributes) (for spans, at least), and already are (or should be) captured by the agent.

Reply:
Thanks for the heads-up. If that's the standard attribute names from the Spec we should use them.

Reply:
I pushed an update renaming this to the standard attributes.